webapp/src/components/searchDialog/searchDialog.tsx

```diff
@@ -36,6 +41,41 @@ const SearchDialog = (props: Props): JSX.Element => {
 
     const emptyResult = results.length === 0 && !isSearching && searchQuery
 
+    const handleUpDownKeyPress = (e: KeyboardEvent) => {
+        if (Utils.isKeyPressed(e, Constants.keyCodes.DOWN)) {
+            e.preventDefault()
+            if (results.length > 0)
+                setSelected(Utils.clamp(selected + 1, 0, results.length - 1))
```

Comment:
```suggestion
                setSelected(((selected + 1) <= (results.length - 1)) ? selected + 1 : selected)
```

e.g. a ternary operator or something else?

Honestly, this might have been tech debt from when I was implementing this PR. I thought initially I would need the up/down functionality to wrap so I was using a modulo operator (`%`) but then realized that's now how Channels switcher worked so I changed to the `clamp` approach and didn't think twice. Happy to remove code though. 🙂

Reply:
Yes, that would work for me, also I would use a bit simpler expression like `(selected+1) < results.length`.

Reply:
Done. 👍🏻